lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/declaration/koproperty/KoPropertyDeclarationForKoValueProviderTest.kt

```diff
@@ -0,0 +1,88 @@
+package com.lemonappdev.konsist.core.declaration.koproperty
+
+import com.lemonappdev.konsist.TestSnippetProvider.getSnippetKoScope
+import org.amshove.kluent.assertSoftly
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
```

Comment:
Are these all options for property? Or maybe I should add something else?

Reply:
We can add 
- - `fun `property-inside-class-has-value`() {`
- `fun `property-inside-class-has-no-value`() {`
- `fun `property-inside-function-has-value`() {`
- `fun `property-inside-function-has-no-value`() {`